.eslintrc

```diff
@@ -32,6 +32,7 @@
     "camelcase": [2, {
       "properties": "never"
     }],
+   "no-console": [0] //allow use of javascript console in tests.
```

Comment:
I agree, we should not allow `console` in our code. If we're using it in tests for debugging we should remove it before the code is merged